repos/system_upgrade/el7toel8/actors/checkrootsymlinks/actor.py

```diff
@@ -0,0 +1,41 @@
+import os
+
+from leapp import reporting
+from leapp.actors import Actor
+from leapp.models import Report
+from leapp.tags import IPUWorkflowTag, ChecksPhaseTag
+
+
+class CheckRootSymlinks(Actor):
+    """
+    Check if the symlinks /bin and /lib are relative, not absolute.
+
+    After reboot, dracut fails if the links are absolute.
+    """
+
+    name = 'check_root_symlinks'
+    consumes = ()
+    produces = (Report,)
+    tags = (IPUWorkflowTag, ChecksPhaseTag)
+
+    def process(self):
+        paths = {
+            '/bin': 'usr/bin',
+            '/lib': 'usr/lib',
+            '/lib64': 'usr/lib64',
+            '/sbin': 'usr/sbin'
+        }
+        absolute_paths = [path for path in paths if os.path.isabs(path)]
```

Comment:
and as well. you cannot work with system in checksphase

Reply:
Done.